language/de.json

```diff
@@ -0,0 +1,372 @@
+{
+	"Preview": "Vorschau",
+	"Gallery": "Galerie",
+	"Generate": "Generieren",
+	"Skip": "Überspringen",
+	"Stop": "Stop",
+	"Input Image": "Eingabebild",
+	"Advanced": "Erweitert",
+	"Upscale or Variation": "Vergrößern oder Abwandeln",
```

Comment:
Suggestion: Vergrössern oder Variieren

Reply:
This suggestion has now been implemented